MessengerOnFileForLinux/scripts/local_scripts/build_msg

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Pamiętaj o:
- skrypt musi mieć taką nazwę byś wolał go używać zamiast ręcznego wpisywania
- clear powinno nie usuwać folder build (tak jak jest teraz)
- skrypt powinien działać z ustawionym PATH -->pierwszy raz używasz to prosi Cię o ustawienie PATH do folderu build, wtedy nie musisz nigdy martwić się gdzie go odpalasz, jednocześnie powinien przenosić Cię do tego folderu, chyba że wyłaczysz to zmienną (np. środowiskową, albo skryptową)
- --make-only powinno być domyślnym działaniem programu, ze względu , że najczęściej to będziesz robił
- skrypt powinien przyjmować wieloargumentowość, czyli możesz puścić i -ut i -cl np.
- zapomniałeś o valgrindzie